src/main/java/org/folio/inventory/dataimport/handlers/actions/CreateInstanceEventHandler.java

```diff
@@ -94,6 +93,12 @@ public CompletableFuture<DataImportEventPayload> handle(DataImportEventPayload d
 
       Context context = EventHandlingUtil.constructContext(dataImportEventPayload.getTenant(), dataImportEventPayload.getToken(), dataImportEventPayload.getOkapiUrl());
       Record targetRecord = Json.decodeValue(payloadContext.get(EntityType.MARC_BIBLIOGRAPHIC.value()), Record.class);
+
+      if (AdditionalFieldsUtil.getValue(targetRecord, TAG_999, SUBFIELD_I).isPresent()) {
```

Comment:
please, add here a check on the "acceptInstanceId" flag existence in the dataImportEventPayload context, and if it is true,
then we will allow incoming record import regardless of 999ffi field existence.

Reply:
Fixed

Reply:
Thank you!